components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -193,6 +194,29 @@ const PrepopulatedEngine brave_yandex =
                        "part={searchTerms}&v=3&sn=5&srv=brave_desktop",
                        PREPOPULATED_ENGINE_ID_YANDEX);
 
+
+const PrepopulatedEngine brave_search = {
+    L"Brave",
+    L":br",
+    "https://cdn.search.brave.com/serp/favicon.ico",
+    "https://search.brave.com/search?q={searchTerms}",
+    "UTF-8",
+    "https://search.brave.com/api/suggest?q={searchTerms}&rich=false",
```

Comment:
Nit. Rich is false by default.
```suggestion
    "https://search.brave.com/api/suggest?q={searchTerms}",
```